scripts/README.md

```diff
@@ -66,6 +66,22 @@ The fleet size should be smaller than or equal to 20, in order to ensure that th
 Please document the displayed bracket-trader addresses. They are required for future withdrawals.
 They can also be retrieved from the created transactions. However, since this is a manual process, it is quite cumbersome to extract them right now.
 
+### Monitoring deployed funds
```

Comment:
```suggestion
### Monitoring Deployed Funds
```

Reply:
This could be turned into an issue, since there is no consistency in the title capitalization of the readme.